### ISSUE TITLE
Migration '0013_auto_20150726_2129.py' is created in app 'cms' after run command 'makemigrations'

### ISSUE THREAD

**Issue by vivazzi** [divio/django-cms#4317]
I use Django==1.8.3, django-cms==3.1.2.
I get the creation of migration after the running comand 'python manage.py makemigrations':
```
Migrations for 'cms':
  0013_auto_20150726_2129.py:
    - Change managers on pageuser
```

**Comment by saeedsq** [comment]
I'm in the same boot.
Django==1.8.3, django-cms==3.1.2
Migrations for 'cms':
  0013_auto_20150730_2004.py:
    - Change managers on pageuser

```python
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import models, migrations
import django.contrib.auth.models

class Migration(migrations.Migration):

    dependencies = [
        ('cms', '0012_auto_20150607_2207'),
    ]

    operations = [
        migrations.AlterModelManagers(
            name='pageuser',
            managers=[
                ('objects', django.contrib.auth.models.UserManager()),
            ],
        ),
    ]
```

**Comment by yakky** [comment]
@vivazzi @saeedsq this has been fixed in support branch and will be released in 3.1.3
